src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/court/VideoLinkBookingService.kt

```diff
@@ -283,6 +322,11 @@ class VideoLinkBookingService(
       )
     }
 
+  private fun hasAnEndDate(it: VideoLinkAppointment): Boolean {
+    val hasEndDate = it.endDateTime != null
+    if (!hasEndDate) log.error("Appointment with id ${it.id} has no end date")
```

Comment:
is this possible? isn't end date a nullable field now?

Reply:
good spot. Fixed now